### PR TITLE
Fix .NET branding in public docs

### PR DESCRIPTION
**Pull request by heaths**
Was an attempt to try fixing issue #6924 but it seems that code may live elsewhere. Figured I might as well update public-facing first-party branding in this repo while I was in here.

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
...
+Core of the APIView tool is the web app developed using ASP.NET and TypeScript. This core module takes care of presenting reviews to users, storing review files and metadata in Azure Storage Account and Cosmos database and process requests from Azure Devops pipelines and respond. We also have language level parsers that converts each language specific artifact into a common json stub file that's known to core APIView web app. So, first step as a contributor is to understand the feature or bug fix you would like to submit and identify the area you would like to contribute to. Language parsers are either added as plugin modules developed in .NEt or developed using corresponding language as command line tool to extract and generate stub file. If change is specific to a language in how langauge specific API details are extracted to stub tokens then change will be at parser level for that language. If change is applicable for all languages then change will most likely be in core APIView web app.
```
Comment (maririos) [inline src/dotnet/APIView/APIViewWeb/CONTRIBUTING.md:17]:
```suggestion
Core of the APIView tool is the web app developed using ASP.NET and TypeScript. This core module takes care of presenting reviews to users, storing review files and metadata in Azure Storage Account and Cosmos database and process requests from Azure Devops pipelines and respond. We also have language level parsers that converts each language specific artifact into a common json stub file that's known to core APIView web app. So, first step as a contributor is to understand the feature or bug fix you would like to submit and identify the area you would like to contribute to. Language parsers are either added as plugin modules developed in .NET or developed using corresponding language as command line tool to extract and generate stub file. If change is specific to a language in how langauge specific API details are extracted to stub tokens then change will be at parser level for that language. If change is applicable for all languages then change will most likely be in core APIView web app.
```

**Review by maririos — APPROVED** [review]

File: packages/python-packages/doc-warden/README.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ the `package_indexing_exclusion_list` array members to enable just this sort of
 
 `package_indexing_traversal_stops` is used during parse of .NET language repos _only_. This is due to how the discovery logic for readme and changelog is implemented for .NET projects. Specifically, readmes for a .csproj are often a couple directories up from their parent .csproj location!
 
-For .net, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
+For .NEt, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
```
Comment (weshaggard) [inline packages/python-packages/doc-warden/README.md:227]:
```suggestion
For .NET, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
```

Comment (heaths) [reply packages/python-packages/doc-warden/README.md:227]: Again? Rats. Let me double check. Seems I released the `shift` key too soon.

Comment (heaths) [reply packages/python-packages/doc-warden/README.md]: Via VSCode, that was the only other one I found. Thanks.

**Review by benbp — APPROVED** [review]

**Review by danieljurek — APPROVED** [review]

**Review by konrad-jamrozik — APPROVED** [review]

**Comment by heaths** [comment]
@konrad-jamrozik the CLI check keeps failing with:

```
Failed to collect coverage from /mnt/vss/_work/1/s/tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/SDKGenerationTaskBase.ts
ERROR: test/unit/DockerTaskEngine.test.ts:1:34 - error TS2307: Cannot find module '@azure-tools/sdk-generation-lib' or its corresponding type declarations
```

Is this a known issue? I can't imagine anything I did affected this. Just override?

**Comment by konrad-jamrozik** [comment]
> @konrad-jamrozik the CLI check keeps failing with:
> 
> ```
> Failed to collect coverage from /mnt/vss/_work/1/s/tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/tasks/SDKGenerationTaskBase.ts
> ERROR: test/unit/DockerTaskEngine.test.ts:1:34 - error TS2307: Cannot find module '@azure-tools/sdk-generation-lib' or its corresponding type declarations
> ```
> 
> Is this a known issue? I can't imagine anything I did affected this. Just override?

@heaths I do not own this logic. Maybe ask on the EngSys General teams channel?

**Comment by heaths** [comment]
Rebasing onto upstream main.

**Comment by weshaggard** [comment]
@raych1 do you have any context on the failing sdk-generation-pipeline issue?

**Pull request merged**